app/uk/gov/hmrc/tariffclassificationfrontend/models/TabIndexes.scala

```diff
@@ -20,9 +20,23 @@ import uk.gov.hmrc.tariffclassificationfrontend.views.CaseDetailPage.{CaseDetail
 
 object TabIndexes {
 
-  private val indexByPage: Map[CaseDetailPage, Int] = Map(TRADER -> 1000, LIABILITY -> 2000 , APPLICATION_DETAILS -> 2000, SAMPLE_DETAILS -> 3000, ATTACHMENTS -> 4000,
-    ACTIVITY -> 5000, KEYWORDS -> 6000, RULING -> 7000, APPEAL -> 8000)
+  private val indexByPage: Map[CaseDetailPage, Int] = Map(TRADER -> 1000, LIABILITY -> 2000 , APPLICATION_DETAILS -> 2000, SAMPLE_DETAILS -> 3000,
+    ATTACHMENTS -> 4000, ACTIVITY -> 5000, KEYWORDS -> 6000, RULING -> 7000, APPEAL -> 8000)
 
   def tabIndexFor: CaseDetailPage => Int = { page => indexByPage.getOrElse(page, 0) }
 
+  private val indexByQueues: Map[String, Int] = Map("my-cases" -> 1000, Queues.gateway.slug -> 2000 , Queues.act.slug -> 3000,  Queues.cap.slug -> 4000,
```

Comment:
If queues were dynamic (stored in Mongo) I don't think this would work, could we not keep my-cases & gateway hardcoded but then iterate through the rest adding 1000 or whatever to the tab index?

Reply:
I'll see what i can do